src/twig/components/article-grid/ArticleGrid.vue

```diff
@@ -1,134 +1,134 @@
+                                    ></div>
+                                </span>
+                            </a>
+                        </div>
+                    </div>
+                </article>
             </div>
-          </div>
-        </article>
-      </div>
+        </div>
     </div>
-  </div>
 </template>
 <script>
-import axios from "axios";
-import moment from "moment";
-import TextGroup from "/src/twig/components/text-group/TextGroup.vue";
+    import axios from 'axios';
+    import moment from 'moment';
+    import TextGroup from '../text-group/TextGroup.vue';
 
-export default {
-  components: {
-    TextGroup,
-  },
-  data() {
-    return {
-      data: [],
-    };
-  },
-  async mounted() {
-    if (this.renderApi) {
-      switch (this.api) {
-        case "president":
-          this.endpoint =
-            "https://news.colby.edu/wp-json/wp/v2/external_post?&tags=577&_embed=1";
-          break;
-      }
+    export default {
+        components: {
+            TextGroup,
+        },
+        data() {
+            return {
+                data: [],
+            };
+        },
+        async mounted() {
+            if (this.renderApi) {
+                switch (this.api) {
+                    case 'president':
+                        this.endpoint =
+                            'https://news.colby.edu/wp-json/wp/v2/external_post?&tags=577&_embed=1';
+                        break;
+                }
 
-      await axios.get(this.endpoint).then((output) => {
-        this.data = output.data.map((item) => {
-          return {
-            title: {
-              rendered: item.title.rendered.replace(/<\/?[^>]+(>|$)/g, ""),
+                await axios.get(this.endpoint).then((output) => {
+                    this.data = output.data.map((item) => {
+                        return {
+                            title: {
+                                rendered: item.title.rendered.replace(/<\/?[^>]+(>|$)/g, ''),
+                            },
+                            'post-meta-fields': {
+                                summary: [
+                                    `${item.content.rendered
+                                        .replace(/<\/?[^>]+(>|$)/g, '')
+                                        .substring(0, 120)}...`,
+                                ],
+                            },
+                            url: item.external_url,
+                            date: moment(item.date).format('MMM DD, YYYY'),
+                        };
+                    });
+                });
+            }
+        },
+        props: {
+            renderApi: {
+                required: true,
+            },
+            api: {
+                type: String,
+                required: false,
+            },
+            border: {
+                type: String,
+                required: false,
             },
-            "post-meta-fields": {
```

Comment:
@GioAceto Does this code ring a bell to you.  Does `post-meta-fields.summary` relate to the changes we made to Article Grid a month or two ago?